src/utils/utils.ts

```diff
@@ -40,95 +41,169 @@ export function diff(a: any[], b: any[]) {
+  // Since that's the name generated from branch name.
+  if (stderr.includes('404: Not Found') || code) {
+    zipName = 'master.zip'
 
-  // NOTE: git version 2.13 and greater are using '--recurse-submodules' instead of '--recurse' attribute to clone submodules
-  shelljs.exec(
-    `cd "${folderPath}" && git clone --recurse${
-      parseInt(gitVersion) > 2130 ? '-submodules' : ''
-    } --depth 1 -b ${gitBranch} ${repoUrl} .`,
-    { silent: true }
-  )
+    const { stdout, stderr, code } = downloadFile(
+      `${repoUrl}${zipPath}${zipName}`
+    )
 
-  deleteGitFolder(folderPath)
+    if (stderr.includes('404: Not Found') || code) {
+      errorCallback(`${repoUrl}${zipPath} is not SASjs repository!`)
 
-  shelljs.rm('-f', [path.join(folderPath, '.gitmodules')])
-
-  if (repoUrl.includes('react-seed-app')) {
-    deleteGitFolder(path.join(folderPath, 'public', 'docs'))
-  } else if (repoUrl.includes('angular-seed-app')) {
-    deleteGitFolder(path.join(folderPath, 'docs'))
+      return
+    }
   }
 
+  const zipWithoutExtension = zipName.replace('.zip', '')
+
+  const zip = new AdmZip(zipName)
+  zip.extractAllTo(`./`, true)
+
+  shelljs.cp('-r', `./*${zipWithoutExtension}/.`, folderPath)
+  shelljs.rm('-rf', [`./*${zipWithoutExtension}`])
+  shelljs.rm('-rf', [`./${zipName}`])
+
+  loadDocsSubmodule(docsUrl, folderPath, fullZipPath)
+  shelljs.rm('-f', [path.join(folderPath, '.gitmodules')])
+
   spinner.stop()
 
   if (installDependencies) {
     spinner.text = 'Installing dependencies...'
     spinner.start()
 
-    shelljs.exec(`cd "${folderPath}" && git init && npm install`, {
+    shelljs.exec(`cd "${folderPath}" && npm install`, {
       silent: true
     })
 
     spinner.stop()
   }
 }
 
-const deleteGitFolder = (folderPath: string) =>
-  shelljs.rm('-rf', path.join(folderPath, '.git'))
+/**
+ * It will download and unzip `sasjs/docs` into newly created folder into `docs` subfolder
+ * @param docsUrl sasjs/docs repo url
+ * @param folderPath full path to the newly created folder that contains the seed app
+ * @param zipPath fixed path of where github puts zip for repo download
+ */
+const loadDocsSubmodule = async (
+  docsUrl: string,
+  folderPath: string,
+  zipPath: string
+) => {
+  let docsFolderPath = `${folderPath}/public/docs` //We first look if docs submodule is inside `public` folder. (react-seed-app for example)
+  if (!(await fileExists(docsFolderPath))) docsFolderPath = `${folderPath}/docs` //If not, we load submodule in root
+
+  downloadFile(`${docsUrl}${zipPath}`)
+
+  const zip = new AdmZip('main.zip')
+  zip.extractAllTo('./', true)
+
+  shelljs.cp('-r', `./*-main/.`, docsFolderPath)
+  shelljs.rm('-rf', [`./*-main`])
+  shelljs.rm('-rf', [`./main.zip`])
+}
+
+function downloadFile(url: string): ShellString {
+  if (isWindows()) {
+    return shelljs.exec(`curl ${url} -LO -f`, { silent: true })
+  } else {
+    return shelljs.exec(`wget ${url}`, { silent: true })
```

Comment:
Not all Mac OS has `wget`, but the `curl` should be available across all platforms.